alterations/alterations_test.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+    "testing"
+    // "fmt"
+    "github.com/OWASP/Amass/v3/alterations"
+)
+
+var subdomain = "fa2ke.test.com"
+
+func check_alterations(generated_subdomains []string, expected_subdomains []string) int {

```

Comment:
In Go, `checkAlterations(generated []string, expected []string) int` is preferred. See this link for some of the preferred naming conventions for Go:

https://talks.golang.org/2014/names.slide#1